data/csp/azure/sle12/packages.yaml

```diff
@@ -23,7 +23,6 @@ packages:
     package:
       - azure-cli
       - lsscsi
-      - python3-azure-sdk
```

Comment:
Should be `python-azure-sdk`

Reply:
I don't see `python-azure-sdk` included at all in SLE 12 so that's why I simply removed `python3-azure-sdk`, which I added accidentally in the first place.

Reply:
Only added it to the 15 images but we should include it in the 12 based images as well since we are shipping the packages and we've sorted out the various issues we had early on.

Reply:
Ok, easy enough to add.